src/overlays/actors/ovl_En_Poh/z_en_poh.c

```diff
@@ -96,123 +150,874 @@ static DamageTable D_80B2F6E0 = {
+    }
+}
+
+void func_80B2D5DC(EnPoh* this) {
+    SkelAnime_ChangeAnimDefaultStop(&this->skelAnime, &D_06000A60);
+    this->actionFunc = func_80B2D628;
+    this->actor.speedXZ = -5.0f;
+}
+
+void func_80B2D628(EnPoh* this, GlobalContext* globalCtx) {
+    if (SkelAnime_FrameUpdateMatrix(&this->skelAnime)) {
+        this->actor.world.rot.y = this->actor.shape.rot.y;
+        func_80B2CB60(this);
+        this->unk_18E = 23;
+    } else {
+        Math_StepToF(&this->actor.speedXZ, 0.0f, 0.5f);
+        this->actor.shape.rot.y += 0x1000;
+    }
+}
+
+void func_80B2D694(EnPoh* this) {
+    SkelAnime_ChangeAnimDefaultRepeat(&this->skelAnime, &D_060015B0);
+    this->unk_192 = BINANG_ROT180(this->actor.world.rot.y);
+    this->actionFunc = func_80B2D6EC;
+    this->actor.speedXZ = 0.0f;
+}
+
+void func_80B2D6EC(EnPoh* this, GlobalContext* globalCtx) {
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    if (Math_ScaledStepToS(&this->actor.world.rot.y, this->unk_192, 0x71C)) {
+        func_80B2CB60(this);
+    }
+
+    if (this->actor.xzDistToPlayer < 200.0f) {
+        func_80B2CD14(this);
+    }
+
+    func_80B2C9B8(this, globalCtx);
+}
+
+void func_80B2D76C(EnPoh* this) {
+    this->unk_18C = 32;
+    this->unk_192 = 0x2000;
+    this->actor.speedXZ = 0.0f;
+    this->actor.world.rot.y = this->actor.shape.rot.y;
+    Audio_PlayActorSound2(&this->actor, NA_SE_EN_PO_DISAPPEAR);
+    Audio_PlayActorSound2(&this->actor, NA_SE_EN_PO_LAUGH);
+    this->colliderCylinder.base.acFlags &= ~AC_ON;
+    this->actionFunc = func_80B2D7D4;
+}
+
+void func_80B2D7D4(EnPoh* this, GlobalContext* globalCtx) {
+    if (this->unk_18C != 0) {
+        this->unk_18C--;
+    }
+
+    this->actor.world.rot.y += this->unk_192;
+
+    if (this->unk_18C < 16) {
+        Math_ScaledStepToS(&this->unk_192, 0, 0x200);
+    }
+
+    func_80B2C9B8(this, globalCtx);
+    this->unk_197 = this->unk_18C * 7.96875f;
+    if (this->unk_18C == 0) {
+        this->unk_190 = Rand_S16Offset(100, 50);
+        this->colliderCylinder.info.bumper.dmgFlags = 0x40001;
+        func_80B2CB60(this);
+    }
+}
+
+void func_80B2D924(EnPoh* this) {
+    this->unk_18C = 0;
+    this->unk_192 = 0x2000;
+    this->actor.speedXZ = 0.0f;
+    Audio_PlayActorSound2(&this->actor, NA_SE_EN_STALKIDS_APPEAR);
+    Audio_PlayActorSound2(&this->actor, NA_SE_EN_PO_LAUGH);
+    this->colliderCylinder.base.acFlags &= ~AC_ON;
+    this->actionFunc = func_80B2D980;
+}
+
+void func_80B2D980(EnPoh* this, GlobalContext* globalCtx) {
+    this->unk_18C++;
+    this->actor.world.rot.y -= this->unk_192;
+    if (this->unk_18C > 15) {
+        Math_ScaledStepToS(&this->unk_192, 0, 0x200);
+    }
+
+    func_80B2C9B8(this, globalCtx);
+    this->unk_197 = this->unk_18C * 7.96875f;
+    if (this->unk_18C == 32) {
+        this->unk_190 = Rand_S16Offset(700, 300);
+        this->unk_18C = 0;
+        this->colliderCylinder.info.bumper.dmgFlags = 0xF7CBFFFE;
```

Comment:
```suggestion
        this->colliderCylinder.info.bumper.dmgFlags = ~0x8340001;
```
I wonder if this would be appropriate?